NumberValidator/Validators/ar/Cbu.cs

```diff
@@ -0,0 +1,125 @@
+// # cbu.py - functions for handling Argentinian CBU numbers
+// # coding: utf-8
+// #
+// # Copyright (C) 2016 Luciano Rossi
+// # Copyright (C) 2022 Sudhanshu Pant
+// #
+// # This library is free software; you can redistribute it and/or
+// # modify it under the terms of the GNU Lesser General Public
+// # License as published by the Free Software Foundation; either
+// # version 2.1 of the License, or (at your option) any later version.
+// #
+// # This library is distributed in the hope that it will be useful,
+// # but WITHOUT ANY WARRANTY; without even the implied warranty of
+// # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// # Lesser General Public License for more details.
+// #
+// # You should have received a copy of the GNU Lesser General Public
+// # License along with this library; if not, write to the Free Software
+// # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+// # 02110-1301 USA
+
+// """CBU (Clave Bancaria Uniforme, Argentine bank account number).
+// CBU it s a code of the Banks of Argentina to identify customer accounts. The
+// number consists of 22 digits and consists of a 3 digit bank identifier,
+// followed by a 4 digit branch identifier, a check digit, a 13 digit account
+// identifier and another check digit.
+
+using System;
+using System.Globalization;
+using NumberValidator.Helpers;
+
+namespace NumberValidator.Validators.AR
+{
+    public class cbu : IValidator
+    {
+        public bool IsValid(string cbu)
+        {
+            try
+            {
+                Validate(cbu);
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        public void Validate(string a)
+        {
+            a = a.Clean();
+
+            ValidateFormat(a);
+        
+        }
+
+        
+        public string Splitter(int i, int j,string cbu)
```

Comment:
Splitter name but how are we splitting? Can we make the purpose/intention of function clear?